code/controllers/subsystem/atoms.dm

```diff
@@ -88,7 +88,8 @@ SUBSYSTEM_DEF(atoms)
 	if(!A)	//possible harddel
 		qdeleted = TRUE
 	else if(!(A.flags_1 & INITIALIZED_1))
-		BadInitializeCalls[the_type] |= BAD_INIT_DIDNT_INIT
+		if (A.type != /mob/dview)  // See /mob/dview for why this is okay
```

Comment:
instead of specifically checking for the type, wouldn't a bitflag of some kind be a bit nicer? (especially if we have any more cases where this would actually be useful in the future)

Reply:
Don't we have a macro that guards against this? I think /new_player or /dead might use it